code/stats/scoring.cpp

```diff
@@ -105,12 +105,10 @@ void parse_rank_tbl()
 		required_string("#End");
 
 		// be sure that all rank points are in order
-#ifndef NDEBUG
```

Comment:
Crap. I was under the impression that I could copy and paste code when I made that change. In that case, disregard my initial comment...

I'll submit a PR for restoring the warning count later today.